.semaphore/semaphore.yml

```diff
@@ -78,8 +78,6 @@ blocks:
                 # until https://github.com/checkstyle/checkstyle/issues/9248
                 # - mvn -e clean install -Pno-validations
                 #    && .ci/validation.sh no-violation-test-configurate
-                - mvn -e --no-transfer-progress clean install -Pno-validations
-                    && .ci/no-exception-test.sh openjdk16-with-checks-nonjavadoc-error
```

Comment:
Why and is there any replacement?

Reply:
replacement at https://github.com/checkstyle/checkstyle/pull/10280/commits/c551021b0cd6bffe44a340b186d73234f5f9b87d#r681365824 per @romani